# Command to robot controller
std_msgs/Header header
string action  # forward, backward, stop, turn_left, turn_right
float32 speed
float32 steering_angle
